src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -214,7 +229,7 @@
       (fn [index segment]
         (if (zero? index)
           {:text "Workspaces" :href "#workspaces"}
-          {:text (replace (js/decodeURIComponent segment) ":" "/")
+          {:text (clojure.string/replace (js/decodeURIComponent segment) ":" "/")
```

Comment:
Cursive was resolving the `replace` as `cljs.core.replace` and flagging it as an error.